### ISSUE TITLE
Changing values in sh_config.lua has no effect.

### ISSUE THREAD

**Issue by imnotliveyet** [vrondakis/Leveling-System#122]
Changing values in sh_config.lua has no effect. Tried to move the "Level: #" text up because it covers the hud, but it didn't move so I tried disabling it. Nothing worked, so I checked to make sure my other changes weren't there and they weren't.

Running the latest version of darkrp on the latest version of gmod

**Comment by thenamesjay** [comment]
Do you perhaps have the github version on the server, but also the workshop version added to the resources?

**Comment by imnotliveyet** [comment]
> Do you perhaps have the github version on the server, but also the workshop version added to the resources?

Yes, originally I did. However, changing that made the addon not load on clients.

Edit: found out about resource.AddWorkshop, trying to figure that out

**Comment by imnotliveyet** [comment]
Can't figure out resource.AddWorkshop as nothing outlines how it is supposed to be used, or even where. Tried garrysmod/lua/autorun/(server/client) and still the addon doesnt load.

**Comment by thenamesjay** [comment]
Skip that, and just do this:
1. Make collection with all addons you want on server
2. Copy collection ID (numbers after the = in the url to collection)
3. At your server's launch options, add "    +host_workshop_collection "IDHERE"    "
4. Ensure the following addon is added to that workshop collection: 
https://steamcommunity.com/sharedfiles/filedetails/?id=2214712098

This should ensure everything works properly. Keep in mind, you need to also ensure the following:
A. That the collection does NOT have the Leveling System addon added to it
and
B. The server itself has the leveling system file from github setup, and has no -'s or _'s.

Should be g2g then.

**Comment by imnotliveyet** [comment]
Followed your steps, I already have client and server side collections and added that mod to the server, while leveling system fixed is on the other one. Still not loading. 
By "no -'s or _'s" do you mean to rename the addon folders so they do not contain those?

**Comment by thenamesjay** [comment]
Correct, you would want to rename the addon folders so they do not contain those.
What do you mean by client and server side collections?
You should just be using one collection and host it for the server, then using that addon I recommended it will only force download client side models and sounds.

**Comment by imnotliveyet** [comment]
I mean I have a collection with *all* mods used by the server for users to subscribe to, then a collection of all mods that aren't in the files for the host collection. 

The problems I have are specifically with the leveling system, all the sounds and models are fine.

**Comment by imnotliveyet** [comment]
Tried resource.AddFile() and it added all of the addons from the folder, LevelingSystem still didn't load
